components/AudioSettings/DecreaseVolume.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect } from "react";
+
+export default function VolumeFade({
+  page,
+  experience,
+  currentVolume,
+  setCurrentVolume,
+  isMuted,
+}) {
+  useEffect(() => {
+    if (currentVolume > 0 && experience.length === 0 && page === 2) {
+      const decreaseVolume = setInterval(() => {
+        setCurrentVolume((previousVolume) => {
+          const newVolume = Math.max(0, previousVolume - 0.01);
+          console.log("decreased:", newVolume);
+          return newVolume;
+        });
+      }, 50);
+      return () => clearInterval(decreaseVolume);
+    }
+  }, [page, experience, currentVolume, setCurrentVolume, isMuted]);
```

Comment:
Please consider whether all dependencies are necessary or if some can be removed to optimise performance (e.g. `isMutated` doesn't influence the effect's behaviour)